delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1074,8 +1084,10 @@ def loadServiceInfo() {
 			def token = sh (script: "curl --silent -X POST -k -v \
 				-H \"Content-Type: application/json\" \
 				 $url \
-				-d \'${payload}\' | $jq -r '.data.token'", returnStdout:true)
-			token = token.replaceAll("[\n\r]", "")
+				-d \'${payload}\'", returnStdout:true)
+			def jsonParser = new groovy.json.JsonSlurper()
```

Comment:
same comment as above.  initialize globally.

Reply:
Tried to initialize  globally but it gave error as ' java.io.NotSerializableException' caused by JsonSlurper.  So initializing needs to be done in a method.